wallet/chainntfns.go

```diff
@@ -293,6 +294,15 @@ func (w *Wallet) onBlockConnected(serializedBlockHeader []byte, transactions [][
 	w.NtfnServer.notifyMainChainTipChanged(chainTipChanges)
 	w.NtfnServer.sendAttachedBlockNotification()
 
+	// Parse vote version from extended bits.  Note that once ExtendedBits
+	// are used for other information as well this will need to be updated
+	// to accomodate.
+	voteVersion := binary.LittleEndian.Uint32(w.VoteBits().ExtendedBits)
+	if voteVersion < blockHeader.StakeVersion {
+		log.Warnf("Old vote version detected (v%v), please update your "+
+			"wallet to the latest version.", voteVersion)
```

Comment:
davecgh suggested to just say to update to latest, instead of specifying which version the network is on (which could confuse the user).

Also voteVersion is the version of the wallet not the connected block.